cloudsql-postgresql-plugin/src/e2e-test/java/io/cdap/plugin/CloudSqlPostgreSqlClient.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.junit.Assert;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+/**
+ * CloudSQLPostgreSQL client.
+ */
+
+public class CloudSqlPostgreSqlClient {
+    public static Connection getCloudSqlConnection() throws ClassNotFoundException, SQLException {
+        Class.forName("org.postgresql.Driver");
+        String database = PluginPropertyUtils.pluginProp("databaseName");
+        String instanceConnectionName = System.getenv("CLOUDSQL_POSTGRESQL_CONNECTION_NAME");
+        String username = System.getenv("CLOUDSQL_POSTGRESQL_USERNAME");
+        String password = System.getenv("CLOUDSQL_POSTGRESQL_PASSWORD");
+        String jdbcUrl = String.format(PluginPropertyUtils.pluginProp("URL"), database, instanceConnectionName, username, password);
```

Comment:
Did you tested the code after adding this URL from plugin properties if its working fine.

Reply:
yes this is working fine.